defaults/main.yml

```diff
@@ -40,3 +40,10 @@ sb_debian_base_supplementary_packages: []
 sb_debian_base_known_hosts:
   - line:  "{{ lookup('pipe', 'ssh-keyscan -t rsa github.com') }}"
     regexp: "^github\\.com"
+
+# Bash history configuration
+sb_debian_base_bash_history: true
+sb_debian_base_bash_history_config:
```

Comment:
> ...if a user define `sb_debian_base_bash_history.histsize` but `sb_debian_base_bash_history` is set to `false` it would not take effect, right?
At task level, it will not modify anything setting `sb_debian_base_history: false` but, I notice that currently is not natively posible to test using role defined variables or facts. An issue like these here: https://github.com/philpep/testinfra/issues/345

In different places people is using Testaid: https://github.com/RebelCodeBase/testaid

I think we could use it as well so in this case we can `assert f.contains('HISTSIZE=6000')` only when `sb_debian_base_bash_history` is set to true.

What do you think?